docs/source/contributing/tests.rst

```diff
@@ -95,24 +94,25 @@ And fixtures to add functionality or spawning behavior:
 - ``bad_spawn``: enables the BadSpawner (a spawner that fails immediately)
 - ``slow_bad_spawn``: enables the SlowBadSpawner (a spawner that fails after a short delay)
 
-For information on using the existing fixtures and creating new ones, refer to the `pytest fixtures documentation <https://pytest.readthedocs.io/en/latest/fixture.html>`_
-
+Refer `pytest fixtures documentation <https://pytest.readthedocs.io/en/latest/fixture.html>`_ 
+to learn how to use fixtures that exists already and also to create new ones
 
 Troubleshooting Test Failures
 =============================
 
 All the tests are failing
 -------------------------
 
-Make sure you have completed all the steps in :ref:`contributing/setup` successfully, and are able to access JupyterHub from your browser at http://localhost:8000 after starting ``jupyterhub`` in your command line.
+Make sure you have completed all the steps in :ref:`contributing/setup` successfully, and
+can launch ``jupyterhub`` from the terminal as well as the web browser
 
 
 Code formatting and linting
 ===========================
 
 JupyterHub has adopted automatic code formatting and linting.
 As long as your code is valid, the pre-commit hook should take care of how it should look.
-You can invoke the pre-commit hook manually at any time with:
+You can invoke the pre-commit hook by hand at any time with :
```

Comment:
No space before punctuation. I'm not sure about the substitution of manually to its synonym "by hand," can you speak to why you've made that change?